lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -9,7 +9,13 @@ def initialize(serializer, options = {})
           super
           @hash = { data: [] }
 
-          if fields = options.delete(:fields)
+          @options[:include] ||= []
+          if @options[:include].is_a?(String)
```

Comment:
Can you explain more about this. It's not related to `type` and `id`, and I also don't understood the `String` verification